app/views/index.scala.html

```diff
@@ -1,11 +1,25 @@
-@(title: String, jsLocation: String, presenceJsLocation: Option[String], clientConfigJson: String)
+@(
+        title: String,
+        jsLocation: String,
+        presenceJsLocation: Option[String],
+        clientConfigJson: String,
+        csrf: Option[String] = None
```

Comment:
should this be optional? I can see only one usage and it's hardcoded as Some

Reply:
Good  point. This was copy+pasted from a project which presumably need the option. I'll change it  here.